package-lock.json

```diff
@@ -60,7 +60,7 @@
         "source-map-support": "^0.5.21",
         "ts-node": "^10.9.2",
         "tsd": "^0.31.2",
```

Comment:
The AC says to update tsd as well

Reply:
Updated and issues fixed. I needed to remove some tests from the slow model tests (since we cannot use namespaced types anymore) but don't think that impacts what that was actually testing.